### DISCUSSION TITLE
Welcome to cargo-logistics Discussions!

### DISCUSSION THREAD

**Discussion by hirwajeaneric** [hirwajeaneric/express-logistics#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hirwajeaneric** [comment]
Hello, It Eric, we shall be using this space to get all the ideas together and easily integrate them in the project

**Reply by Denyse1** [reply]
yes brother
